lib/elixir/lib/enum.ex

```diff
@@ -1403,6 +1403,8 @@ defmodule Enum do
   Invokes `fun` for each element in the collection passing that element and the
   accumulator `acc` as arguments. `fun`'s return value is stored in `acc`.
   The first element of the collection is used as the initial value of `acc`.
+  If you wish to use another value for `acc`, use `Enumerable.reduce/3`.
+  This function won't reduce a one-element list.
```

Comment:
It will reduce a one-element list, but the given anonymous function won't be called.